src/java.base/share/classes/jdk/internal/classfile/impl/BufWriterImpl.java

```diff
@@ -175,6 +189,11 @@ public void writeIntBytes(int intSize, long intValue) {
         }
     }
 
+    public void skip(int skipSize) {
```

Comment:
Conceptually I would prefer to make such an API return an int for skipped index so we can more easily patch later. But that is really optional.